website/developer-docs/docs/writing-documentation.md

```diff
@@ -9,50 +9,61 @@ the documentation is easy to read and uses similar phrasing.
 
 Requirements:
 
--   Node 16 (or greater)
+-   Node.js 16 (or greater)
 
 The documentation site is situated in the `/website` folder of the authentik GitHub repository.
 
 The site is built using npm, below are some useful make commands:
 
--   Install: `make website-install` (Needed for any of the other tasks)
--   Formatting: `make website-lint-fix` or `make website` (Run this before committing)
--   Live editing: `make website-watch` (For real time viewing of changes)
+-   **Installation**: `make website-install`
 
+    This command is rquired before running any of the following commands, and after upgrading any dependencies.
+
+-   **Formatting**: `make website` or `make website-lint-fix`
+
+    Run this command before committing, otherwise the build will fail with an error about linting.
```

Comment:
I'd change this a bit to show the reason why we do automated formatting (to keep everything in the same format, use consistent markdown syntax, etc) instead of just "warning" users of the potential negative outcome